src/systems/systems/src/hydraulic/mod.rs

```diff
@@ -1 +1,1985 @@
+/// Normal Qty:
+/// -----------
+/// Blue: 6.5L (1.7 US Gal)
+/// Yellow: 12.5L (3.3 US Gal)
+/// Green: 14.5L (3.8 US Gal)
+///
+/// Loops
+/// ------------------------------------------
+/// Max loop volume - green: 100L 26.41gal including reservoir
+/// Max loop volume - yellow: 75L 19.81gal including reservoir
+/// Max loop volume - blue: 50L 15.85gal including reservoir
+///
+/// EDP (Eaton PV3-240-10C/D/F (F is neo)):
+/// ------------------------------------------
+/// 37.5 GPM max (100% N2)
+/// 3750 RPM
+/// 3000 PSI
+/// Displacement: 2.40 in3/rev
+///
+///
+/// Electric Pump (Eaton MPEV3-032-EA2 (neo) MPEV-032-15 (ceo)):
+/// ------------------------------------------
+/// Uses 115/200 VAC, 400HZ electric motor
+/// 8.45 GPM max
+/// 7600 RPM at full displacement, 8000 RPM at no displacement
+/// 3000 PSI
+/// Displacement: 0.263 in3/rev
+///
+///
+/// PTU (Eaton Vickers MPHV3-115-1C):
+/// ------------------------------------------
+/// 2987 PSI
+///
+/// Yellow to Green
+/// ---------------
+/// 34 GPM (130 L/min) from Yellow system
+/// 24 GPM (90 L/min) to Green system
+/// Maintains constant pressure near 3000PSI in green
+///
+/// Green to Yellow
+/// ---------------
+/// 16 GPM (60 L/min) from Green system
+/// 13 GPM (50 L/min) to Yellow system
+/// Maintains constant pressure near 3000PSI in yellow
+///
+///
+/// RAT PUMP (Eaton PV3-115):
+/// ------------------------------------------
+/// Max displacement: 1.15 in3/rev, 18.85 mL/rev
+/// Normal speed: 6,600 RPM
+/// Max. Ov. Speed: 8,250 RPM
+/// Theoretical Flow at normal speed: 32.86 gpm, 124.4 l/m
+///
+///
+/// Equations:
+/// ------------------------------------------
+/// Flow (Q), gpm:  Q = (in3/rev * rpm) / 231
+/// Velocity (V), ft/s: V = (0.3208 * flow rate, gpm) / internal area, sq in
+/// Force (F), lbs: F = density * area * velocity^2
+/// Pressure (P), PSI: P = force / area
+/// PressureDelta = VolumeDelta / Total_uncompressed_volume * Fluid_Bulk_Modulus
+///
+///
+/// Hydraulic Fluid: EXXON HyJet IV
+/// ------------------------------------------
+/// Kinematic viscosity at 40C: 10.55 mm^2 s^-1, +/- 20%
+/// Density at 25C: 996 kg m^-3
+///
+/// Hydraulic Line (HP) inner diameter
+/// ------------------------------------------
+/// Currently unknown. Estimating to be 7.5mm for now?
+///
+///
+/// Actuator Force Simvars
+/// -------------------------------------------
+/// ACCELERATION BODY X (relative to aircraft, "east/west", Feet per second squared)
+/// ACCELERATION BODY Y (relative to aircraft, vertical, Feet per second squared)
+/// ACCELERATION BODY Z (relative to aircraft, "north/south", Feet per second squared)
+/// ROTATION VELOCITY BODY X (feet per second)
+/// ROTATION VELOCITY BODY Y (feet per second)
+/// ROTATION VELOCITY BODY Z (feet per second)
+/// VELOCITY BODY X (feet per second)
+/// VELOCITY BODY Y (feet per second)
+/// VELOCITY BODY Z (feet per second)
+/// WING FLEX PCT (:1 for left, :2 for right; settable) (percent over 100)
+///
```

Comment:
**question**: Was this intended as a module level comment? Then prefix with `//!` and put it at the top of the file. Otherwise remove one level of `/` as to not confuse the docs generation.